packages/core/src/app/address/googleAutocomplete/GoogleAutocompleteFormField.tsx

```diff
@@ -53,7 +53,7 @@ const GoogleAutocompleteFormField: FunctionComponent<GoogleAutocompleteFormField
             id: getAddressFormFieldInputId(name),
             'aria-labelledby': labelId,
             placeholder: useFloatingLabel ? ' ' : placeholder,
-            labelText: useFloatingLabel ? labelContent : null,
+            'data-labeltext': useFloatingLabel ? labelContent : null,
```

Comment:
This is to avoid a React warning for passing `labelText` as an invalid input field prop.

Reply:
I believe we only need to make a change inside `Autocomplete` where `inputProps` is passed to `input`. Its parent / wrapper element could keep using `labelText` property name.

Reply:
I revert it back to `labelText` and created `validInputProps` in `Autocomplete.tsx` to filter out `labelText`.
https://github.com/bigcommerce/checkout-js/blob/f83558975b831aee6216da02119e3521acca27f8/packages/core/src/app/ui/autocomplete/Autocomplete.tsx#L50